src/lib.rs

```diff
@@ -95,19 +95,19 @@
 #![allow(clippy::field_reassign_with_default)]
 #![allow(clippy::arc_with_non_send_sync)]
 
-pub mod request;
+pub mod backoff;
```

Comment:
For backoff, is is a good idea to use [backon](https://github.com/Xuanwo/backon/) instead?